pkg/injector/annotations/annotations.go

```diff
@@ -47,6 +47,9 @@ const (
 	KeyReadinessProbePeriodSeconds      = "dapr.io/sidecar-readiness-probe-period-seconds"
 	KeyReadinessProbeThreshold          = "dapr.io/sidecar-readiness-probe-threshold"
 	KeySidecarImage                     = "dapr.io/sidecar-image"
+	KeySidecarProfile                   = "dapr.io/sidecar-profile"
+	KeySidecarDropCapability            = "dapr.io/sidecar-drop-capability"
```

Comment:
should this be `sidecar-drop-capabilities`?

Reply:
Base on the comment [here](https://github.com/dapr/dapr/issues/5916#issuecomment-1433879612), I removed both the annotations and implemented a way to drop or preserve capabilities through helm chart